samples/SwiftDemoApp/SwiftDemoApp/Samples.swift

```diff
@@ -13,11 +13,11 @@
  * limitations under the License.
  */
 
-import Foundation
+import UIKit
 
 class Samples {
-  class func loadSamples() -> [Any] {
-    return [ newDemo(controller: ClusteringViewController.self,
+    class func loadSamples() -> [[String: Any]] {
```

Comment:
Nit: please update spacing back to 2 spaces. You can also update your default to 2 spaces in Xcode under preferences.

![Screen Shot 2020-07-13 at 9 36 12 AM](https://user-images.githubusercontent.com/463186/87329761-4fe5af00-c4ec-11ea-9aad-059b9d0aff85.png)

Reply:
Cool will do it 👍